setup.py

```diff
@@ -43,4 +48,13 @@
         'Topic :: Documentation',
         'Topic :: Software Development :: Documentation',
     ],
+    install_requires= [
+        'Sphinx>=1.4.1',
+    ],
+    cmdclass = {
+        'compile_catalog': babel.compile_catalog,
```

Comment:
When babel is installed, its setup.py commands should work automatically, without your changes to `setup.py`.